[2025-06-18 15:01:53,515] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2022-04-29T00:00:00+00:00 [queued]>
[2025-06-18 15:01:53,548] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2022-04-29T00:00:00+00:00 [queued]>
[2025-06-18 15:01:53,548] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:01:53,552] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:01:53,553] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:01:53,629] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29T00:00:00+00:00
[2025-06-18 15:01:53,637] {standard_task_runner.py:52} INFO - Started process 5732 to run task
[2025-06-18 15:01:53,648] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2022-04-29T00:00:00+00:00', '--job-id', '10481', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp9cabhks1', '--error-file', '/tmp/tmpd3qt6b1t']
[2025-06-18 15:01:53,653] {standard_task_runner.py:77} INFO - Job 10481: Subtask second_task
[2025-06-18 15:01:53,877] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2022-04-29T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:01:54,019] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2025-06-18 15:01:54,020] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:01:54,022] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:01:54,041] {bash.py:169} INFO - Output:
[2025-06-18 15:01:54,044] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:01:54,045] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:01:54,212] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20220429T000000, start_date=20250618T150153, end_date=20250618T150154
[2025-06-18 15:01:54,446] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:01:54,498] {local_task_job.py:146} INFO - Task exited with return code 0
